cedar-policy-core/src/authorizer.rs

```diff
@@ -386,16 +375,29 @@ impl Default for Authorizer {
     }
 }
 
-#[derive(Debug, Clone, Default)]
+#[derive(Debug, Clone)]
 struct EvaluationResults<'a> {
     satisfied_permits: Vec<&'a Policy>,
     satisfied_forbids: Vec<&'a Policy>,
     global_deny_policies: HashSet<PolicyID>,
-    all_warnings: Vec<String>,
+    errors: Vec<(PolicyID, EvaluationError)>,
     permit_residuals: Vec<Policy>,
     forbid_residuals: Vec<Policy>,
 }
 
+impl Default for EvaluationResults<'_> {
+    fn default() -> Self {
+        Self {
+            satisfied_permits: vec![],
+            satisfied_forbids: vec![],
+            global_deny_policies: HashSet::new(),
+            errors: vec![],
+            permit_residuals: vec![],
+            forbid_residuals: vec![],
+        }
+    }
+}
```

Comment:
This is the same as what we would previously have gotten with the auto-derived `Default`; can we keep using that?

Reply:
In some version of my edits, the automatic derivation of `Default` failed, which is why I implemented it manually. Forgot to go back and check whether it was still required (it's not).